lib/core-complete.sh

```diff
@@ -4360,10 +4392,15 @@ function ble/widget/complete {
     fi
   fi
   if ((cand_count==0)); then
+    local complete_limit_reached=
     ble/complete/generate-candidates-from-opts "$opts"; local ext=$?
     if ((ext==148)); then
       return 148
-    elif ((ext!=0||cand_count==0)); then
+    fi
+    if [[ $complete_limit_reached ]]; then
+      ble/term/visible-bell "complete: limit reached"
```

Comment:
I would like to use `ble/widget/.bell` rather than the hardcoded `ble/term/visible-bell`. `ble/widget/.bell` is a configurable bell by the options `bleopt edit_vbell` and `bleopt edit_abell` (maybe it is quicker than words to see the actual implementation by `type ble/widget/.bell`).

Also, could you prepend `[[ :$opts: != *:no-bell:* ]] &&` before the bell similarly to the subsequent call of `ble/widget/.bell`. I would like to suppress the bell when the internal complete option `no-bell` is specified.